tsconfig.base.json

```diff
@@ -71,6 +71,7 @@
         "libs/shared/db/type-cacheable/src/index.ts"
       ],
       "@fxa/shared/error": ["libs/shared/error/src/index.ts"],
+      "@fxa/shared/error/error": ["libs/shared/error/src/lib/error.ts"],
```

Comment:
The additional route alias is required to import the generic error message into both the client and server sides of the app

Reply:
I think it'd be a good idea, that if a library needs to export items specifically for use on the client or on the server, the alias and ideally the barrel file name, should clearly indicate this.

This doesn't need to be done as part of this PR. Lets make the changes as part of a polish PR.